src/main/ngapp/accio/src/app/ImageBlob.ts

```diff
@@ -0,0 +1,40 @@
+
+/**
+ *  Contains all relevant information when posting a blob to the server.
+ *  Constructors are made based on the required information, anything not
+ *    passed through the constructor is set to an empty string or false.  
+ *    
+ */
+export class ImageBlob {
+  // Required params
+  projectId: string;
+  imageName: string;
+  mode: string;
+
+  image: any;
+  parentImageName: string;
+  newImageName: string;
+  tags: string;
```

Comment:
Are these expected to be comma separated?  Something like this is good to clarify so people knew to the codebase can find out.